### PR TITLE
Add DBML target

### LINKED ISSUE #1431: [Feature Request] Integrate with DBML diagrams
First of all, thanks for your awesome work!

We often use https://dbdiagram.io/ to design database architecture. 

This tool is based on a domain specific language called DBML which has been released as open source: https://github.com/holistics/dbml

Another open source tool in node/Typescript is available to convert this DBML diagrams into SVG: https://github.com/softwaretechnik-berlin/dbml-renderer/

This tool is normally called from the command line, but they also provide an `api.ts` which allows to use it as a library.

Would it be possible to integrate in kroki?

### PR DESCRIPTION
**Pull request by bobzomer**
Try to implement DBML support as proposed in https://github.com/yuzutech/kroki/issues/1431.

I think most of the work is done (I hope so), but it still fails with error 500 during smoke tests.

Closes #1431

### CODE REVIEW

File: dbml/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const encoding = 'utf-8'
+let data
+
+async function convert () {
+  const source = data.toString(encoding)
+  if (source === '') {
+    return
+  }
+  try {
+    console.log(render)
```
Comment (Mogztter) [inline dbml/index.js:13]:
```suggestion
```

**Comment by Mogztter** [comment]
I found a typo, I believe that's why it was not working: [yuzutech/kroki@`64154c2` (#1432)](https://github.com/yuzutech/kroki/pull/1432/commits/64154c266450e6d1389d3b4f9a53cd524c777c18)

**Comment by bobzomer** [comment]
> I found a typo, I believe that's why it was not working: [yuzutech/kroki@`64154c2` (#1432)](https://github.com/yuzutech/kroki/pull/1432/commits/64154c266450e6d1389d3b4f9a53cd524c777c18)

Wonderful!

**Comment by Mogztter** [comment]
Thanks for your contribution 🤗

**Comment by bobzomer** [comment]
> Thanks for your contribution 🤗

Thanks for your correction and your awesome project!

**Comment by bobzomer** [comment]
Now that it's merged, could you release it so that we can start using it? More specifically we are using the self-hosted docker version.

**Pull request merged**